mne/time_frequency/tfr.py

```diff
@@ -505,7 +505,8 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     from ..viz.utils import _setup_vmin_vmax
 
     copy = baseline is not None
-    data = rescale(data, times, baseline, mode, copy=copy)
+    if mode is not None:
+        data = rescale(data, times, baseline, mode, copy=copy)
```

Comment:
As far as the code here is concerned, I'm saying that the check that is currently here should be moved to `rescale` since rescale already has related checks.

As far as the code in the example is concerned, we should do whatever makes the example look most reasonable. You could try setting `mode` to different values and see what works best.

Come to think of it, we should probably make it throw an error if `baseline is not None and mode is None` to avoid people thinking they're baseline correcting when they're not. And those checks should be done in `rescale`.